projects/components/src/datagrid/filters/datagrid-numeric-filter.component.ts

```diff
@@ -0,0 +1,91 @@
+/*!
+ * Copyright 2019 VMware, Inc.
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+import { Component, Host, OnInit } from '@angular/core';
+import { DatagridFilter, FilterConfig } from './datagrid-filter';
+import { ClrDatagridFilter } from '@clr/angular';
+import { FormBuilder, FormGroup } from '@angular/forms';
+import { FilterBuilder } from '../../utils/filter-builder';
+
+enum FormFields {
+    from = 'from',
+    to = 'to',
+}
+
+/**
+ * Numeric filter UI widget has only single configuration. So there are no properties in addition to FilterConfig
+ */
+export type DatagridNumericFilterConfig = FilterConfig<[number, number]>;
+
+@Component({
+    selector: 'vcd-dg-numeric-filter',
+    template: `
+        <div [formGroup]="formGroup" class="clr-row">
+            <div class="clr-col-6">
+                <label class="clr-control-label">From:</label>
```

Comment:
Clicking the label doesn't take you to the input right now? That's also not accessible to readers. I know I asked you to remove a hard coded id. There are usually two solutions:

* Make the input be inside the label (simpler, use when possible)
* Dynamically create an ID to avoid duplicate IDs